core/lib/libtomcrypt/src/ciphers/aes/aes_modes_armv8a_ce_a64.S

```diff
@@ -466,6 +467,7 @@ ENTRY(ce_aes_ctr_encrypt)
 	adds            x6, x6, #1              /* increment BE ctr */
 	rev             x7, x6
 	ins             v4.d[1], x7
+	cbnz            x9, .Lctrcarrydone      /* ignore overflow */
```

Comment:
ignore overflow for 64-bit counter value.

Reply:
Yeah that's the intended behaviour. Do you suggest to update comment?

Reply:
Yes please.